plugins/feeds/public/benkow_rat.py

```diff
@@ -0,0 +1,70 @@
+import csv
+import logging
+from dateutil import parser
+from datetime import datetime, timedelta
+
+from core.observables import Url, Ip
+from core.feed import Feed
+from core.errors import ObservableValidationError
+
+
+class BenkowTrackerRat(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=12),
+        "name": "BenkowTrackerRat",
+        "source": "http://benkow.cc/export_rat.php",
+        "description": "This feed contains known Malware C2 servers",
+    }
+
+    def update(self):
+
+        since_last_run = datetime.utcnow() - self.frequency
+
+        resp = self._make_request()
+        reader = csv.reader(resp.content.strip().splitlines(), delimiter=';', quotechar='"')
+        for line in reader:
+            if line[0] == 'id':
+                return
+
+            id, family, url, ip, first_seen, _ = line
```

Comment:
Please keep the parsing logic in `update` to a minimum - everything should go in `analyze()` (only keep the first_seen check in `update` and put the rest in `analyze`)